src/Aks/Aks/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added parameter `-NodePodSubnetID` for `New-AzAksCluster`, `-PodSubnetID` for `New-AzAksNodePool`
+* Bumped API version to 2023-02-01
```

Comment:
this is a hybrid module, not all cmdlet got API version upgraded, please remove this line

Reply:
Removed.